wallet-new/src/Cardano/Wallet/Kernel/Submission.hs

```diff
@@ -368,7 +394,7 @@ tickSlot :: Slot
 tickSlot currentSlot ws =
     let (allEvents, schedule) = scheduledFor currentSlot (ws ^. wsState . wssSchedule)
         scheduledCandidates = filterNotConfirmed (allEvents ^. seToSend <> nursery schedule)
-        localPending = ws ^. wsState . wssPendingSet
+        localPending = ws ^. wsState . wssPendingMap
         topSorted  = topsortTxs toTx scheduledCandidates
```

Comment:
We want to put a comment here that it's useful to sort all the tx at the wallet (not account) granularity as that helps in case of `inter-account` transactions, as effectively it makes the system a bit more rigid (cross-reference Ryan's ticket).